schema/spacewalk/spacewalk-schema-upgrade

```diff
@@ -10,12 +10,11 @@ use Getopt::Long ();
 use Pod::Usage ();
 
 my $MIN_PG_DB_VERSION = 130001;
-my $MAX_PG_DB_VERSION = 149999;
+my $MAX_PG_DB_VERSION = 169999;
 my $MIN_PG_DB_VERSION_STRING = "13";
-my $MAX_PG_DB_VERSION_STRING = "14";
-my $OS_VERSION_CHECK = 15.4;
-my $OS_VERSION_MIN_DB_VERSION = 140000;
-my $OS_VERSION_WANTED_DB_VERSION = "14";
+my $MAX_PG_DB_VERSION_STRING = "15";
```

Comment:
These DB versions are not consistent. `MAX_PG_DB_VERSION` being 169999 allows pgsql 16. But the string `MAX_PG_DB_VERSION_STRING`, which is actually presented to the user says max version 15.


```suggestion
my $MAX_PG_DB_VERSION_STRING = "16";
```